src/site/content/en/blog/maskable-icon/index.md

```diff
@@ -0,0 +1,99 @@
+    Different platform specific shapes
+  </figcaption>
+</figure>
+
+Luckily, there's a well-defined and [standardized](https://w3c.github.io/manifest/#icon-masks) "minimum safe zone" that all platforms respect. The important parts of your icon, such as your logo, should be within a circular area in the center of the icon with a radius equal to 40% of the icon width. The outer 10% edge may be cropped.
+
+You can check which parts of your icons land within the safe zone with Chrome DevTools. With your Progressive Web App open, launch DevTools and navigate to the "Application" panel. In the "Icons" section, you can choose to "Show only the minimum safe area for maskable icons". Your icons will be trimmed so that only the safe area is visible. If your logo is visible within this safe area, you're good to go.
+
+<figure class="w-figure w-figure--center">
+  <img src="devtools.png" class="w-screenshot" alt="Applications panel in DevTools displaying PWA icons with edges cropped">
+  <figcaption class="w-figcaption">
+    The Applications panel
+  </figcaption>
+</figure>
+
+If you want to preview your maskable icon in other shapes it may appear in on Android, I've created a tool called [Maskable.app](https://maskable.app/). Open an icon, then Maskable.app will let you try out various shapes and sizes, and you can share the preview with others on your team.
+
+## How do I adopt maskable icons?
+
+If you want to create a maskable icon based off your existing icon, you can use the [Maskable.app Editor](https://maskable.app/editor). Upload your icon, adjust the color and size, then export the image.
+
+<figure class="w-figure w-figure--center">
+  <img src="maskable-app-editor.png" class="w-screenshot" alt="Maskable.app Editor screenshot">
+  <figcaption class="w-figcaption">
+    Creating icons in Maskable.app Editor
+  </figcaption>
+</figure>
+
+Once you've created a maskable icon image and tested it out in DevTools, you'll need to update your [Web App Manifest](https://developers.google.com/web/fundamentals/web-app-manifest) to point to the new assets. The Web App Manifest provides information about your web app in a JSON file, and includes an [`icons` array](https://developers.google.com/web/fundamentals/web-app-manifest#icons).
+
+With the inclusion of maskable icons, a new property value has been added for image resources listed in a Web App Manifest. The `purpose` field tells the browser how your icon should be used. By default, icons will have a purpose of `"any"`. These icons will be resized on top of a white background on Android.
+
+Maskable icons should use a different purpose: `"maskable"`. This indicates that an image is meant to be used with icon masks, giving you more control over the result. This way, your icons will not have a white background. You can also specify multiple space-separated purposes (for example, `"any maskable"`), if you want your maskable icon to be used without a mask on other devices.
+
+```json
+{
+  ...
```

Comment:
```suggestion
  …
```
We use ellipsis character rather than 3 periods because assistive technology announces the ellipsis character more accurately